src/Neo.VM/Utility.cs

```diff
@@ -27,6 +27,20 @@ static Utility()
             StrictUTF8.EncoderFallback = EncoderFallback.ExceptionFallback;
         }
 
+        public static bool TryGetString(this byte[] byteArray, out string? value)
+        {
+            try
+            {
+                value = StrictUTF8.GetString(byteArray);
+                return true;
+            }
+            catch
+            {
+                value = default;
+                return false;
+            }
+        }
+
```

Comment:
If core isn't using this move to test project.

Reply:
@Jim8y what about this?

Reply:
all set